contracts/solidity/ERC777/ERC777ContractAccount.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.20;
+
+import "./ERC777Token.sol";
+import "@openzeppelin/contracts/interfaces/IERC1820Registry.sol";
+
+
+contract ERC777ContractAccount {
+
+    /**
+     * @dev Error when call instruction is not successful
+     */
+    error CallReverted();
+
+    /**
+     * @dev hashes of the necessary interfaces
+     */
+    bytes32 internal constant _TOKENS_SENDER_INTERFACE_HASH = keccak256("ERC777TokensSender");
+    bytes32 internal constant _TOKENS_RECIPIENT_INTERFACE_HASH = keccak256("ERC777TokensRecipient");
+
+    /**
+     * @notice currently ERC1820 is not supported on Hedera networks. Therefore, this implementation requires a manual ERC1820Registry deployment
```

Comment:
Maybe say deployed vs supported. 
Supported read like there was a technical limitation
```suggestion
     * @notice currently ERC1820 is not deployed on Hedera networks. Therefore, this implementation requires a manual ERC1820Registry deployment
```

Reply:
resolved! and yeah that sounds better thanks for the suggestion!